core/src/test/java/org/axonframework/queryhandling/backpressure/BufferBackpressureTest.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2010-2018. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ *
+ */
+
+package org.axonframework.queryhandling.backpressure;
+
+import org.axonframework.queryhandling.UpdateHandler;
+import org.junit.*;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.mockito.Mockito.*;
+
+/**
+ * Tests for {@link BufferBackpressure}.
+ *
+ * @author Milan Savic
+ */
+public class BufferBackpressureTest {
+
+    @SuppressWarnings("unchecked")
```

Comment:
What 'unchecked' warning are you suppressing?

Reply:
`mock(UpdateHandler.class)` is suppressed because it's of raw type. Found a way to lose `@SuppressWarnings` :)